cmake/BLTPrivateMacros.cmake

```diff
@@ -8,6 +8,52 @@ include(CMakeParseArguments)
 ## Internal BLT CMake Macros
 
 
+##-----------------------------------------------------------------------------
+## blt_determine_scope(TARGET <target>
```

Comment:
This macro is nice.

It looks like you are handling the case to make this function case-insensitive.
It might be a bit confusing for the documentation to list the scopes using lower case since cmake actually uses upper case.  

Could you please either switch the case in the documentation to upper case, or add a note that the function is case sensitive, but returns the scope in ``OUT`` as upper case?